project_national/panel_upgrades.yml

```diff
@@ -0,0 +1,1320 @@
+      
+      # Other Fuel as backup
+      - &fed_min_ashp_exist_backup_shared_ducts_60_of
+        option: HVAC Heating Efficiency|Dual-Fuel ASHP, SEER 15.05, 8.82 HSPF, Integrated Backup, Supplemental Backup Sizing, 60% AFUE Other Fuel, 0F-40F switchover band
+        apply_logic:
+          - HVAC Heating Efficiency|Fuel Furnace, 60% AFUE
+          - HVAC Heating Type And Fuel|Other Fuel Fuel Furnace        
+        costs: *costs_ashp_ducted_8-82_HSPF
+        lifetime: *lifetime_HP
+      - &fed_min_ashp_exist_backup_shared_ducts_76_of
+        option: HVAC Heating Efficiency|Dual-Fuel ASHP, SEER 15.05, 8.82 HSPF, Integrated Backup, Supplemental Backup Sizing, 76% AFUE Other Fuel, 0F-40F switchover band
+        apply_logic:
+          - HVAC Heating Efficiency|Fuel Furnace, 76% AFUE
+          - HVAC Heating Type And Fuel|Other Fuel Fuel Furnace        
+        costs: *costs_ashp_ducted_8-82_HSPF
+        lifetime: *lifetime_HP
+      - &fed_min_ashp_exist_backup_shared_ducts_80_of
+        option: HVAC Heating Efficiency|Dual-Fuel ASHP, SEER 15.05, 8.82 HSPF, Integrated Backup, Supplemental Backup Sizing, 80% AFUE Other Fuel, 0F-40F switchover band
+        apply_logic:
+          - HVAC Heating Efficiency|Fuel Furnace, 80% AFUE
+          - HVAC Heating Type And Fuel|Other Fuel Fuel Furnace        
+        costs: *costs_ashp_ducted_8-82_HSPF
+        lifetime: *lifetime_HP
+      - &fed_min_ashp_exist_backup_shared_ducts_92-5_of
+        option: HVAC Heating Efficiency|Dual-Fuel ASHP, SEER 15.05, 8.82 HSPF, Integrated Backup, Supplemental Backup Sizing, 92.5% AFUE Other Fuel, 0F-40F switchover band
+        apply_logic:
+          - HVAC Heating Efficiency|Fuel Furnace, 92.5% AFUE
+          - HVAC Heating Type And Fuel|Other Fuel Fuel Furnace        
+        costs: *costs_ashp_ducted_8-82_HSPF
+        lifetime: *lifetime_HP
+
+      # Electricity as backup
+      - &fed_min_ashp_exist_backup_shared_ducts_electricity
+        option: HVAC Heating Efficiency|ASHP, SEER 15.05, 8.82 HSPF, Max Load, Supplemental Backup Sizing
+        apply_logic:
+          - HVAC Heating Efficiency|Electric Furnace, 100% AFUE        
+        costs: *costs_ashp_ducted_8-82_HSPF
+        lifetime: *lifetime_HP
+
+      ## MSHP non-ducted heating ##
+      - &fed_min_mshp_exist_backup
+        option: HVAC Heating Efficiency|MSHP, SEER 14.5, 8.33 HSPF, Separate Backup, Supplemental Backup Sizing, Max Load 
+        apply_logic: 
+          - HVAC Has Shared System|None
+          - HVAC Has Ducts|No
+          - not: HVAC Heating Type And Fuel|Electricity MSHP
+        costs: *costs_mshp_ductless_8-33_HSPF
+        lifetime: *lifetime_HP
+
+    #### 1.4 High-Efficiency Heat Pump With Electric Backup, 2 options  ####
+      - &1_high_eff_ashp_ducted
+        option: HVAC Heating Efficiency|ASHP, SEER 20, 11 HSPF, CCHP, Max Load, Supplemental Backup Sizing
+        apply_logic:
+          - HVAC Has Ducts|Yes
+          - not: Heating Fuel|None
```

Comment:
Can we update both the yaml and intent doc to install heat pump even for Heating Fuel|None? Removing this would mean we are installing HP for homes that did not have heating but have CAC (hence have ducts), which still makes sense.

Reply:
done